services/scan/src/precinct_scanner_app.ts

```diff
@@ -91,7 +91,7 @@ export async function buildPrecinctScannerApp(
   const { store } = workspace;
 
   // Load interpreter configuration from the store on startup
-  if (store.getElectionDefinition()) {
+  if (store.getElectionDefinition() && store.getPrecinctSelection()) {
```

Comment:
Since there are now a bunch of repeated checks that the precinct selection is configured before configuring the interpreter, we could move them inside the `configureInterpreter` function (along with the check for election definition).

Reply:
I like that, will do one last refactor. Seems like a potential foot gun later if we don't.